Week2/README.md

```diff
@@ -178,34 +191,34 @@ To learn more about this topic, check out the following:
 Let’s say we wanted to get a list of those students and the details of their teacher. This would be a perfect fit for an inner join, since an inner join returns records at the intersection of the two tables.
 
 ```sql
-SELECT s.first_name, s.last_name, s.gender, s.grade, t.full_name
-FROM students s
-INNER JOIN teachers t
-ON s.teacher_number = t.teacher_number
+SELECT students.first_name, students.last_name, students.gender, students.grade, teachers.full_name
```

Comment:
It is a bit verbose.
In the previous query, students know how to use aliases to improve (cut-short) the length of queries

Reply:
I made it verbose on purpose to demonstrate the use of aliases right after this example. I didn't see any explanation on aliases and heard questions about it today.

Reply:
Perhaps we could add explanation about aliases here (And in lesson plans if it is not there). PR for that would be super-welcome.

Reply:
On line 200 of this same file is my addition to explain aliases ;)